61_rotate_ist.py

```diff
@@ -0,0 +1,30 @@
+# Definition for singly-linked list.
+# class ListNode:
+#     def __init__(self, val=0, next=None):
+#         self.val = val
+#         self.next = next
+class Solution:
+    def rotateRight(self, head: Optional[ListNode], k: int) -> Optional[ListNode]:
+        if not head: return head
```

Comment:
It would be better if you wrote the if statement and the body on separate lines. (Best coding practices)